.gitlab-ci.yml

```diff
@@ -64,6 +64,31 @@ test:extra-tools:release-tool:
     # Run release-tool unit tests.
     - python3 -m pytest extra/test_release_tool.py
 
+test:docs:production:
+  image: tiangolo/docker-with-compose
+
+  services:
+    - docker:19.03.5-dind
+
+  before_script:
+    - apk add --no-cache bash git openssl pwgen python3 jq
+    - git config --global user.name "user"
+    - git config --global user.email "user@example.com"
+
+  script:
+    - echo "testing docs"
+
+  except:
+    variables:
+      - $RUN_TESTS_STAGING == "true"
+
+  only:
+    refs:
+      - merge_requests
+    changes:
+      - docker-compose*
```

Comment:
I would suggest putting `demo` and `production/*` also in this list.

Reply:
(I'm changing this pr a lot and playing around still, so this might disappear):

I think it's a good idea but I'd like like to get an ACK from @lluiscampos

Reply:
This is now obsolete; but if we were to keep the `changes`, `production` should have been here (`demo` I think would have been irrelevant, even though harmless).